WordPress/gutenberg
# Social Links: Add Patreon, Telegram, and Tiktok icons

## Description

Adds three new social icons to the Social Links block

- Patreon — Closes #25008
- Telegram — Closes #24572
- Tiktok — Closes #21701


## How has this been tested?

- Confirm the icons can be added within social links block
- Confirm the colors look correct for the different style variants

## Screenshots

![image](https://user-images.githubusercontent.com/45363/96024575-c30f0200-0e08-11eb-8045-8b5608e05d7f.png)


## Types of changes

- Adds new SVG files for the icons
- Adds new variants for the icons
- Adds new CSS rules for the icons
- Adds PHP for server-side rendering of the icons
